### ISSUE TITLE
Add option to stop chown'ing files

### ISSUE THREAD

**Issue by Suika** [sahib/rmlint#370]
As the title says, add an option to stop the chown'ing of files.
Since I don't always work with my user and then use root, it makes no sense to chown the files.

Also, 'Paranoia mode' section in tutorial.html needs fixing. There are two -PP

**Comment by sahib** [comment]
You can already do that. See the ``-T`` option in the man page:

```bash
# Search for every lint type, but bad UIDs/GIDs.
$ rmlint -T 'all,-badids'
```

> Also, 'Paranoia mode' section in tutorial.html needs fixing. There are two -PP

Fixed (62b6a783028c5b47f50f3ed64bbe8efcc3d7fe5f).

**Comment by Suika** [comment]
Ah, now I get it how -T works. Thanks.
